src/pages/documentation/adobe-journey-optimizer/iam/public-classes.md

```diff
@@ -14,10 +14,6 @@ Identifier of the `Message`. This value matches the Message Execution ID assigne
 
 <TabsBlock orientation="horizontal" slots="heading, content" repeat="2"/>
```

Comment:
Update repeat count to 1 in the TabsBlock because of removed Android Tabs
```
<TabsBlock orientation=“horizontal” slots=“heading, content” repeat=“1”/>
```